src/Cabal/Parser.hs

```diff
@@ -51,7 +52,8 @@ compatParseGenericPackageDescription bs = case runParseResult (parseGenericPacka
                                                 Nothing    -> pok
                                                 Just perrs -> force ([], Left (Just sv, perrs))
  where
-   go v = msum $ [ goV220 | v < mkVersion [2,3]  ]
+   go v = msum $ [ goV300 | v < mkVersion [3,1]  ]
```

Comment:
@hvr < 3.1 correct here?

Reply:
hrm... actually... something's not right here... I'll take care of it

Reply:
For the record, I now wrote upgrading instructions for next time at https://github.com/haskell-hvr/cabal-parsers/blob/master/cabal-parsers.cabal#L14-L49 :-)